config.yaml

```diff
@@ -20,3 +20,44 @@ sdk:
       #
       # Environment variable: OTEL_SERVICE_NAME
       service.name: !!str "unknown_service"
+  # Configure the tracer provider.
+  tracer_provider:
+    # List of span processors. Each span processor has a name and args used to configure it.
+    span_processors:
+      # Add a batch span processor.
+      #
+      # Environment variable: OTEL_BSP_*, OTEL_TRACES_EXPORTER
+      - name: batch
+        # Configure the batch span processor.
+        args:
+          # Sets the delay interval between two consecutive exports.
+          #
+          # Environment variable: OTEL_BSP_SCHEDULE_DELAY
+          schedule_delay: 5000
+          # Sets the maximum allowed time to export data.
+          #
+          # Environment variable: OTEL_BSP_EXPORT_TIMEOUT
+          export_timeout: 30000
+          # Sets the maximum queue size.
+          #
+          # Environment variable: OTEL_BSP_MAX_QUEUE_SIZE
+          max_queue_size: 2048
+          # Sets the maximum batch size.
+          #
+          # Environment variable: OTEL_BSP_MAX_EXPORT_BATCH_SIZE
+          max_export_batch_size: 512
+          # Sets the exporter. Exporter has a name and args used to configure it.
+          #
+          # Environment variable: OTEL_TRACES_EXPORTER
+          exporter:
+            name: zipkin
```

Comment:
I've added zipkin here but would also be useful to demonstrate otlp and jaeger. Perhaps I add additional `batch processors with to demonstrate otlp and jaeger, but omit the comments and configuration on repeated fields? I.e. exclude `schedule_delay`, `export_timeout`, `max_queue_size`, `max_export_batch_size`.

Reply:
Is it allowed to configure another exporter under the same batch processor? that would be ideal when I want to setup one or more exporters with the same processor config.

Reply:
Currently you'd need to define a second batch processor and add an exporter.

Reply:
I can see why that is the case. I think this is a confusing experience for people who work with collector and SDK config. Am I correct to assume that if args are not specified the corresponding env is considered before the default value?

Reply:
> Is it allowed to configure another exporter under the same batch processor? that would be ideal when I want to setup one or more exporters with the same processor config.

That's a nice idea and is more efficient than having multiple batch processors. Java has `Multi{Signal}Exporter` like [MultiSpanExporter](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/MultiSpanExporter.java) which do just this. Unfortunately this concept doesn't appear in the spec. 

If a multi exporter concept was added to the spec, you could imagine doing something like:
```
  tracer_provider:
    span_processors:
      - name: batch
        args:
          exporter:
            name: multi
            args:
              exporters:
               - name: otlp
                  args: ...
               - name: jaeger
                  args: ....
```
So much nesting..  being able to reference [configured exporters by name](https://github.com/MrAlias/otel-schema/issues/12#issuecomment-1325253265) starts to look appealing.

> Am I correct to assume that if args are not specified the corresponding env is considered before the default value?

If I understand correctly, you're asking about what happens when both file config and environment variable configuration are present, and the file config doesn't specify some piece of configuration. Configuration priority and merging is an open question we need to address.

Reply:
>That's a nice idea and is more efficient than having multiple batch processors. Java has Multi{Signal}Exporter like [MultiSpanExporter](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/MultiSpanExporter.java) which do just this. Unfortunately this concept doesn't appear in the spec.

py, and js have a similar `MultiSpan{Processor/Exporter}` concept. And I believe some variation of multi processor/exporter is used in all SDKs. I wonder if it can be added to the spec or if it's an SDK implementation detail which doesn't need to be spec'd.

 > Configuration priority and merging is an open question we need to address
 
 👍